MorphicServer/AliveEndpoint.cs

```diff
@@ -32,6 +34,14 @@ namespace MorphicServer
     [OmitMetrics]
     public class AliveEndpoint: Endpoint
     {
+
+        public AliveEndpoint(IHttpContextAccessor contextAccessor, ILogger<AliveEndpoint> logger): base(contextAccessor)
```

Comment:
Can this be done in the parent class (Endpoint.cs) by default for all endpoints?

Reply:
The dependency injection requires each class to have a constructor that takes the injected objects.  The base class handles the context stuff, but c# requires subclasses to explicitly implement their own constructors even if they just call base(contextAccessor).
The way I’ve done ILogger on the client is to make it specific to the class, like ILogger<AliveEndpoint>, which automatically tags log messages with AliveEndpoint

Not sure if that’s compatible with how the Serilog stuff is setup